app/controllers/addresses_controller.rb

```diff
@@ -1,13 +1,16 @@
 class AddressesController < ApplicationController
   def update
-    selected_answers_store = SelectedAnswerStore.new(session)
-
-    property_reference = update_params[:property_reference]
+    if property_reference.empty?
```

Comment:
Would this make sense as a validation on a form object - or on the Address model?

Reply:
(maybe, but this works fine)
This should be `blank?` though